### ISSUE TITLE
ovos-audo fails to start when no piper-tts model is found

### ISSUE THREAD

**Issue by builderjer** [OpenVoiceOS/ovos-audio#24]

**Comment by builderjer** [comment]
This is confirmed to be a piper-tts issue.  if I use ovos-tts-plugin-server, still no internet, it falls back to mimic1 correctly

**Comment by ChanceNCounter** [comment]
Probably with our Piper plugin, rather than with Piper, but time will tell.

**Comment by builderjer** [comment]
I agree, I will link an issue there also

**Comment by emphasize** [comment]
Always nice - especially in our case - to know about the versions to be able to reproduce.
This is a pretty dated `ovos-audio` alpha (.2a15) that pretty likely was conflicting with the piper plugin version.
Anyhow, I wouldn't prioritize it with high priority

**Comment by NeonDaniel** [comment]
Should this be transferred to the plugin? Seems like the validator should have caused the plugin to fail before the exception was raised here

**Comment by JarbasAl** [comment]
More generically, the solution here is to also add the runtime requirements, so plugins may indicate they need internet before load. Started it on skills but the goal is to do it on all plugins

**Comment by builderjer** [comment]
I think that it should check for a local model before requiring internet

**Comment by JarbasAl** [comment]
> I think that it should check for a local model before requiring internet

Yeah it only downloads if it doesn't exist, in this case the runtime requirements would do that check before reporting if it's needed or not
